### ISSUE TITLE
Issue with Hardware Addresses

### ISSUE THREAD

**Issue by platyple** [flozz/rivalcfg#105]
Hi, me again :)
I have a Rival 300 Fade Edition but rivalcfg doesn't recognise it (same error as below). 
I checked lsusb and the address is shown there as 1038:1716. Forcing rivalcfg to address that device gets rid of the error but doesn't let me change the settings (`rivalcfg: error: no such option: -c`).
Forcing it to use the Fade Edition profile but my device address gets me this:
```
W: The following mouse was found but rivalcfg was not able to open it:
  * mouse: SteelSeries Rival 300 CS:GO Fade Edition (1038:1394)
  * error: Unable to find the requested device: 1038:1394:00
```
This seems like it would be a quick fix but I have no idea how
Again, Thanks in advance!

**Comment by flozz** [comment]
You are right, it is in the database:

![Capture d’écran de 2020-04-25 17-25-31](https://user-images.githubusercontent.com/971438/80283678-cfe45000-8719-11ea-842b-afc3a64f5eb6.png)

Can you give me its exact name (the one outputed by the `lsusb` command). Have you a link to an online shop that sell this mouse?

This will allow me to add this mouse to rivalcfg :)

**Comment by platyple** [comment]
the line in lsusb outputs is `Bus 003 Device 002: ID 1038:1716 SteelSeries ApS SteelSeries Rival 300 CS:GO Fade Edition`

Unfortunately it was a gift so i can't tell you where it came from exactly, but it should've come from Valve's official store (https://valvestore.forfansbyfans.com/) around mid 2017.

Let me know if i need to plug in my Linux machine for something else you need, lol

**Comment by flozz** [comment]
So, I added the support of your mouse in a branch. Can you test it and tel me if it work?

To test, first clone this repository and go to the test branch:

    git clone https://github.com/flozz/rivalcfg
    cd rivalcfg
    git checkout rival300-csgo-fade-edition-stm32

You should already have all dependencies installed as you aleady installed rivalcfg. So just test with the following command (from the project dir):

    python -m rivalcfg --list

You should see your mouse in the list with `(plugged)` at the end of the line.

you can now test various commands (you will probably need to run it as root because the udev rules file will not be updated yet)

:)

**Comment by platyple** [comment]
Now I'm getting `W: The following mouse was found but rivalcfg was not able to open it` along with `__main__.py: error: no such option: -C`, unfortunately not much of an improvement.
This is a two day old install of Fedora so I don't know what would be interfering with it...

**Comment by flozz** [comment]
Have you tried to run the command as root?

**Comment by platyple** [comment]
Sorry to leave you hanging, had a bunch of stuff come up IRL. It vörks when run as root, thank you for your help :)

**Comment by flozz** [comment]
Ok, so if it work as root, I can merge the branch and release a new version :D

(you can update the `/etc/udev/rules.d/99-steelseries-rival.rules` file with the one from the branch and then run `udevadm trigger`. This will allow you to run rivalcfg with a standard user :) )

**Comment by flozz** [comment]
→ released in **rivalcfg v3.11.0** :)
